app/views/admin_merchants/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>New Admin Merchant</h1>
+<div id=create_admin_merchant>
+  <%= form_with url: "/admin/merchants", method: :post, local: true do |form| %>
```

Comment:
might be better to use the new form_with :model approach. Mixing the types caused issues in a previous section because they need different private: params syntax. We can refactor if this is needed.